libs/saksbehandling-common/src/main/kotlin/beregning/BeregningDTO.kt

```diff
@@ -53,5 +52,6 @@ data class AvkortingGrunnlagKildeDto(
 data class AvkortetYtelseDto(
     val fom: LocalDate,
     val tom: LocalDate?,
+    val avkortingsbeloep: Int,
```

Comment:
det var ikke sånn at vi skulle vise "ytelse før avkorting" også her, eller hentes den fra beregning direkte?

Reply:
Ligger nå i domeneklasse men det ble ikke etterspurt i visningen for "beregning etter avkorting". Det vil vel være nyttig å ha feltet i dto også senere i andre sammenhenger. F.eks brev.  Men tenker at vi kan legge til når vi trenger det.